modules/developer_manual/pages/core/acceptance-tests.adoc

```diff
@@ -39,11 +39,11 @@ Here's an example configuration:
 ----
 default:
   autoload:
-    '': %paths.base%/../features/bootstrap
+    '': '%paths.base%/../features/bootstrap'
```

Comment:
Recent YML parsing requires these to be in quotes.
Put the quotes here so that somebody who copies from here gets joy.